Update learn-DOT + learn-nft (#6679)
w3f/polkadot-wiki: docs/learn/learn-DOT.md

---
title: DOT Token Overview
description: Overview of Polkadot's native token DOT, its uses, denominations, and how to obtain it.
---

## What is DOT?

DOT is Polkadot's native token, similar to BTC for Bitcoin or ETH for Ethereum.

### The Planck Unit

The smallest unit of account balance on Polkadot SDK-based blockchains is Planck, named after the [Planck Length](https://en.wikipedia.org/wiki/Planck_length). DOT equals 10<sup>10</sup> Planck, while Kusama's KSM equals 10<sup>12</sup> Planck.

> ### Polkadot Denominations

Reviewer: I suggest using tabs here and changing the section title to "Polkadot and Kusama Denominations"

| MilliKSM (mKSM) | 9              | 10<sup>9</sup> Planck  | 0.001000000000 KSM |
| KSM             | 12             | 10<sup>12</sup> Planck | 1.000000000000 KSM |

## Uses of DOT

DOT has three main functions:

1. **Governance**: DOT holders control network governance, including fees, parachain management, and upgrades. Learn more on the [governance page](./learn-polkadot-opengov.md).

> 2. **Staking**: DOT is staked to secure the network via [Nominated Proof of Stake](./learn-staking.md). Staking discourages malicious behavior through [slashing](./learn-offenses.md).

Reviewer: ```suggestion
2. **Staking**: DOT is staked to secure the network via [Nominated Proof of Stake](./learn-staking.md).
```

> 3. **Resource Purchase**: DOT is used to reserve [parachain identifiers](../general/glossary.md#paraid) and [Coretime](../general/glossary.md#coretime) for rollups.

Reviewer: ```suggestion
3. **Resource Purchase**: DOT is used to purchase [Coretime](../general/glossary.md#coretime) and reserve [parachain identifiers](../general/glossary.md#paraid) for rollups.
```

> DOT is also transferrable and can fund Polkadot-related projects via the Treasury.

Reviewer: ```suggestion
The DOT token is transferable; a portion of its inflation is diverted to the Treasury, which can fund Polkadot-related projects via OpenGov referenda.
```
